crates/uv/src/settings.rs

```diff
@@ -1398,6 +1401,11 @@ impl PipInstallSettings {
             dry_run,
             constraints_from_workspace,
             overrides_from_workspace,
+            modifications: if flag(exact, inexact).unwrap_or(false) {
```

Comment:
I think this should default to false. I've made the change, and the tests pass for me locally.

Reply:
Thanks, my bad.